pages/documentation/getting-started-developers/phase-two.md

```diff
@@ -1,45 +1,61 @@
 
-We’ve developed `uswds-compile`, a tool hosted on GitHub, to help teams copy USWDS static assets and transform USWDS Sass into browser-readable CSS. [Step 2](#step-2-install-uswds-compile) describes how to install this tool. When you use `uswds-compile`, we will apply Autoprefixer to all compiled code.
+## Introducing uswds-compile
+
+We’ve developed `uswds-compile`, a tool [hosted on GitHub](https://github.com/uswds/uswds-compile), to help teams compile USWDS Sass. This tool provides simple Gulp 4 functions that copy USWDS static assets into your project and transform USWDS Sass into browser-readable CSS. Additionally, `uswds-compile` applies Autoprefixer to all compiled code to make sure your CSS is browser-compatible.
+
+We recommend using `uswds-compile` as a quick way to get up and running with USWDS Sass compilation.
 
 {: .site-note }
-**Note:** If you want `uswds-compile` to set up a Sass entry point for you, skip directly to [Step 2](#step-2-install-uswds-compile). We provide Step 1 for context: why you need a Sass entry point, what it needs to include, and how to set one up manually.
+Note: One of the benefits of using `uswds-compile` is that it will set up a Sass entry point for you. If you plan on using this tool, you can safely skip the next section about Sass entry points and jump straight to the [step-by-step instructions](#step-1-install-uswds-compile) that explain how to install and use `uswds-compile`.
+
+If you prefer to set up Sass compilation yourself, the next thing you'll need to do is set up (or edit) your Sass entry point.
+
+## About Sass entry points
+
+A project often has many Sass files, but typically, there’s a single file that serves as the root — the “homepage” of the Sass — that links out to the others. This root file is also known as the “Sass entry point.” The Sass entry point is the most important stylesheet file in your project because it tells the compiler what source files make up your Sass codebase. Often, a project's Sass entry point is named something like `index.scss` or `styles.scss`.
+
+### Setting up USWDS in your Sass entry point
 
-## Step 1: Set up your project’s Sass entry point
+If your project does not yet have a Sass entry point, create a file called `index.scss` or `styles.scss`.
 
-A project often has many Sass files, but typically, there’s a single file that serves as the root — the “homepage” of the Sass — that links out to the others. This root file is also known as the “Sass entry point.” The Sass entry point is the most important stylesheet file in your project because it tells the compiler what source files make up your Sass codebase.
+When your entry point is ready, you'll need to load USWDS Sass. Your Sass entry point will need to do the following three tasks in this order:
```

Comment:
I have added some language ("For your entry point to complete these tasks") immediately after these tasks are identified to help transition the user. Let me know if you think it needs more. 

I am also wondering if `USWDS Sass` sounds a bit like its own product, rather than just referring to the Sass files inside USWDS. Do you see this too?